docs/source/creating-interfaces/html-with-idom.rst

```diff
@@ -0,0 +1,84 @@
+HTML With IDOM
+==============
+
+In a typical Python-base web application the resonsibility of defining the view along
+with its backing data and logic are distributed between a client and server
+respectively. With IDOM, both these tasks are centralized in a single place. This is
+done by allowing HTML interfaces to be constructed in Python. Let's consider the HTML
+sample below:
+
+.. code-block:: html
+
+    <h1>My Todo List</h1>
+    <ul>
+        <li>Build a cool new app</li>
+        <li>Share it with the world!</li>
+    </ul>
+
+We can recreate the same thing with IDOM using functions attached to the
+:mod:`idom.html` module. These function share the same names as their corresponding HTML
+tags. For example, the `<h1/>` element above has a similarly named :func:`idom.html.h1`
+function. Given this we can write the following:
+
+.. testcode::
+
+    from idom import html
```

Comment:
Can these two code blocks be placed side-by-side in a markdown table? Should be two sections `Traditional HTML` and `IDOM`